### PR TITLE
'currentColor' is now inherited as a keyword

### PR DESCRIPTION
**Pull request by faceless2**
currentColor now inherits as a keyword, but border-color-11 and 12 did not reflect this (note this will cause a failure in Webkit, which still inherits as a color value)

See comment from @svgeesus from the first time I tried to apply this: https://github.com/web-platform-tests/wpt/pull/10755#pullrequestreview-190356109

### CODE REVIEW

File: css/CSS2/borders/border-color-012.xht [hunk 1]
```diff
@@ -6,10 +6,10 @@
...
   <link rel="help" href="http://www.w3.org/TR/CSS21/box.html#border-color-properties" />
   <link rel="match" href="border-color-012-ref.xht" />
-
+  <meta name="assert" content="The initial value of border-color is 'currentColor', which (as of css-color-3) is inherited as a keyword." />
   <style type="text/css">
-   .test { display: block; color: green; border: none; }
-   .test > .inner { border-color: inherit; border-style: solid; color: red; }
+   .test { display: block; color: red; border: none; }
+   .test > .inner { border-color: inherit; border-style: solid; color: green; }
    .test > .inner > .text { color: green; }
```
Comment (frivoal) [inline css/CSS2/borders/border-color-012.xht]: The changes are correct, but the `.test > .inner > .text { color: green; }` statement is no longer useful, and should be removed
```suggestion
```

**Review by frivoal — CHANGES_REQUESTED** [review]

File: css/CSS2/borders/border-color-011.xht [hunk 2]
```diff
@@ -6,11 +6,12 @@
...
+  <link rel="help" href="https://drafts.csswg.org/css-color/#resolving-color-values" />
   <link rel="match" href="border-color-011-ref.xht" />
-
+  <meta name="assert" content="The initial value of border-color is 'currentColor', which (as of css-color-3) is inherited as a keyword." />
   <style type="text/css">
-   .test { display: block; color: green; border: none; }
-   .test .inner { border-color: inherit; border-style: solid; color: red; }
+   .test { display: block; color: red; border: none; }
+   .test .inner { border-color: inherit; border-style: solid; color: green; }
    .test .inner .text { color: green; }
```
Comment (frivoal) [inline css/CSS2/borders/border-color-011.xht]: The changes are correct, but the `.test .inner .text { color: green; }` statement is no longer useful, and should be removed
```suggestion
```

Comment (faceless2) [reply css/CSS2/borders/border-color-011.xht]: Very true! Done.

**Review by frivoal — APPROVED** [review]

**Comment by frivoal** [comment]
@Ms2ger @gsnedders @foolip Not sure what's making the Azure checks fail. Can any of you wpt wizzards help debug this? This PR is otherwise good to go.

**Comment by frivoal** [comment]
(in the meanwhile, Azure pipeline failures don't block merging, so I'm merging)

**Comment by foolip** [comment]
Pardon the trouble, this was because of https://github.com/web-platform-tests/wpt/issues/21691 which has been fixed.

**Pull request merged**